src/WpfMath/TexFormulaParser.cs

```diff
@@ -247,6 +247,29 @@ private TexFormula Parse(SourceSpan value, ref int position, bool allowClosingDe
             return formula;
         }
 
+        private static TexFormula ParseChars(SourceSpan value, string textStyle)
```

Comment:
Let's find a better name for this. `ParseRawText`? Or maybe even `ConvertRawText`, since we aren't actually parsing anything?